src/ray/core_worker/actor_manager.cc

```diff
@@ -96,9 +96,8 @@ std::pair<std::shared_ptr<const ActorHandle>, Status> ActorManager::GetNamedActo
     std::ostringstream stream;
     stream << "Failed to look up actor with name '" << name << "'. This could "
            << "because 1. You are trying to look up a named actor you "
-           << "didn't create. 2. The named actor died. 3. The actor hasn't "
```

Comment:
3 is not the case anymore.

Reply:
It also exposes implementation detail. Note that after we fix https://github.com/ray-project/ray/issues/20129, the 2 will be deleted from here too.